modules/nw-ovn-kubernetes-live-migration-about.adoc

```diff
@@ -0,0 +1,83 @@
+
+////
+[IMPORTANT]
+====
+Migrating to or from the OVN-Kubernetes network plugin is not supported for managed OpenShift cloud services such as {product-dedicated}, Azure Red Hat OpenShift(ARO), and Red Hat OpenShift Service on AWS (ROSA).
+====
+////
+
+include::snippets/sdn-deprecation-statement.adoc[]
+
+The following sections provide more information about the live migration method.
+
+[id="supported-platforms-live-migrating-ovn-kubernetes"]
+== Supported platforms when using the live migration method
+
+The following table provides information about the supported platforms for the live migration type.
+
+.Supported platforms for the live migration method
+[cols="1,1", options="header"]
+|===
+| Platform              | Live Migration
+
+| Bare metal hardware             |&#10003;
+| Amazon Web Services (AWS)       |&#10003;
+| Google Cloud Platform (GCP)     |&#10003;
+| {ibm-cloud-name}                |&#10003;
+| Microsoft Azure                 |&#10003;
+| {rh-openstack-first}            |&#10003;
+| VMware vSphere                  |&#10003;
+| AliCloud                        |&#10003;
+| Nutanix                         |&#10003;
+|===
+
+[id="considerations-live-migrating-ovn-kubernetes-network-provider_{context}"]
+== Considerations for live migration to the OVN-Kubernetes network plugin
+
+Cluster administrators should consider the following before using the live migration method to the OVN-Kubernetes network plugin:
+
+* The live migration procedure is unsupported for clusters with OpenShift SDN multitenant mode enabled. You must disable multitenant mode before beginning the process.
+
+* When the live migration procedure begins, multitenant isolation is permanently disabled. Multitenant isolation is only supported on OpenShift SDN.
+
+* Egress routers block the live migration process. This feature must be removed before beginning the live migration process.
+
+* During the live migration, multicast, egress IP addresses, and egress firewalls are temporarily disabled. They can be migrated from OpenShift SDN to OVN-Kubernetes after the live migration process has finished.
+
+* Although a rollback procedure is provided, the migration is intended to be a one-way process. For users that want to rollback to OpenShift-SDN, migration from OpenShift-SDN to OVN-Kubernetes must have succeeded. Users can follow the same procedure below to migrate to the OpenShift SDN network plugin from the OVN-Kubernetes network plugin.
+
+* The live migration not supported on HyperShift clusters.
+
+* New nodes should not be created during the live migration because both CNIs would try to setup the new node.
+
+* OpenShift SDN does not support IPsec. After migration, cluster administrators can enable IPsec.
+
+* OpenShift SDN does not support IPv6. Consequently, live migration to OVN-Kubernetes with IPv6 is not supported. Cluster administrators can enable IPv4, IPv6, or dual-stack if necessary after migration is completed.
```

Comment:
Cluster administrators can enable IPv6 single-stack, or dual-stack if necessary after migration is completed.

Reply:
Changed to 

```suggestion
* OpenShift SDN does not support IPv6. Consequently, live migration to OVN-Kubernetes with IPv6 is not supported. Cluster administrators can enable IPv4, IPv6 single-stack, or dual-stack if necessary after migration is completed.
```

Do you want the mentioning of IPv4 removed entirely?

Reply:
I was wrong. You cannot migrate from IPv4 single stack to IPv6 single stack. So here we shall only mention dual-stack. We don't neet to mention IPv4 single stack, once the migration is completed, the cluster is already IPv4 single stack.

Reply:
@pliurh how is this? 

* OpenShift SDN does not support IPv6. Consequently, live migration to OVN-Kubernetes with IPv6 is not supported. Cluster administrators can enable dual-stack if necessary after migration is completed.